FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/FlowMode/InformixMode/SetUp/content.txt

```diff
@@ -0,0 +1,3 @@
+!|dbfit.InformixTest|
+
+|Connect|localhost:16466|dbfit|DBFIT|dbfit_ifx|
```

Comment:
I think at some point we started converting these to `ConnectUsingFile` in order to be able to easier switch to our custom database instance.